config/src/main/java/com/alibaba/nacos/config/server/service/notify/AsyncNotifyService.java

```diff
@@ -138,52 +132,52 @@ private void executeAsyncInvoke() {
                         // get delay time and set fail count to the task
                         asyncTaskExecute(task);
                     } else {
-                        HttpGet request = new HttpGet(task.url);
-                        request.setHeader(NotifyService.NOTIFY_HEADER_LAST_MODIFIED,
-                                String.valueOf(task.getLastModified()));
-                        request.setHeader(NotifyService.NOTIFY_HEADER_OP_HANDLE_IP, InetUtils.getSelfIp());
+                        Header header = Header.newInstance();
+                        header.addParam(NotifyService.NOTIFY_HEADER_LAST_MODIFIED, String.valueOf(task.getLastModified()));
+                        header.addParam(NotifyService.NOTIFY_HEADER_OP_HANDLE_IP, InetUtils.getSelfIp());
                         if (task.isBeta) {
-                            request.setHeader("isBeta", "true");
+                            header.addParam("isBeta", "true");
+                        }
+                        AsyncNotifyCallBack asyncNotifyCallBack = new AsyncNotifyCallBack(task);
+                        try {
+                            restTemplate.get(task.url, header, Query.EMPTY, String.class, asyncNotifyCallBack);
```

Comment:
Should we modified this `restTemplate.get` method, if any error when execute get, call the callback.onError in the` restTemplate.get` rather than throw exception?

Reply:
Ok, very good advice